automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -261,11 +261,18 @@ def product_upgrade(
     if product not in products:
         print('Product name should be one of {0}'.format(', '.join(products)))
         sys.exit(1)
+
+    for env_var in ('SAT_IMAGE', 'SAT_HOST', 'CAP_IMAGE', 'CAP_HOST'):
```

Comment:
I would suggest to do something like the following:

```python
missing_vars = [
    var for var in ('SAT_IMAGE', 'SAT_HOST', 'CAP_IMAGE', 'CAP_HOST')
    if var not in os.environ
]
if missing_vars:
    print('The following environment variable(s) must be set: {0}.'.format(', '.join(missing_vars)))
    sys.exit(1)
```

With the above you will know all the missing variables at once and will help setting the missing ones.